packages/loader/container-loader/src/container.ts

```diff
@@ -1930,6 +1931,7 @@ export class Container
 
 		this.mc.logger.sendPerformanceEvent({
 			eventName: `ConnectionStateChange_${ConnectionState[value]}`,
+			...props,
```

Comment:
Let's JSON.stringify these and put it in `details` or `reason` field

Reply:
Oh, never mind, I gave other feedback above - I think it'll be better to actually pass the error around and pass the error to the logger here.